src/Modules/Main.lua

```diff
@@ -233,8 +234,6 @@ end
 
 function main:Shutdown()
 	self:CallMode("Shutdown")
-
-	self:SaveSettings()
```

Comment:
I believe this was removed for testing an committed by mistake?  We want settings to save on shutdown, unless you have a reason not to.

Sidenote: This might be the cause of people getting settings wiped out on occasion, if SimpleGraphic calls for a shutdown and fails to let the Lua code do its job before terminating the process.

Reply:
Removing this was intentional. It's now handled by the actual save button at line 749. It was rather irritating having a save button not actually do what it claims while debugging.

Reply:
@zao  pointed out on Discord that this is still necessary due to miscellaneous changes that can occur during a session that aren't currently being saved anywhere else, so I restored it.